SwiftFlowTests/StoreTests.swift

```diff
@@ -77,6 +77,16 @@ class StoreSpecs: QuickSpec {
                 expect(subscriber.receivedStates[subscriber.receivedStates.count - 1]
                     .testValue).to(equal(20))
             }
+            
```

Comment:
Trailing Whitespace Violation: Lines should not have trailing whitespace. (trailing_whitespace)